pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -2467,97 +2468,41 @@ status:
 	}
 }
 
-func TestExpandMountPath(t *testing.T) {
```

Comment:
was this test case (ExpandMountPath) accidentally deleted? I see it on main but not on this branch

Reply:
Thanks for the catch. Yes, this was accidentally deleted.